### PR TITLE
Auth0 transfer

### PR DESCRIPTION
**Pull request by tehnools**

### CODE REVIEW

**Comment by williamparry** [comment]
Could you please fix the merge conflicts?

**Comment by kge89** [comment]
It would good to match your indentation to those of the file(s) so that the diff doesn't show modifications to the files simply because the number of spaces used for indentation has changed. It makes it hard to see what the "real" changes are.

**Comment by tehnools** [comment]
> It would good to match your indentation to those of the file(s) so that the diff doesn't show modifications to the files simply because the number of spaces used for indentation has changed. It makes it hard to see what the "real" changes are.

What are your settings? How many spaces?

File: api/Controllers/CasesController.cs [hunk 1]
```diff
@@ -10,6 +11,70 @@
 
 namespace api.Controllers
 {
+<<<<<<< HEAD
```
Comment (kge89) [inline api/Controllers/CasesController.cs]: Looks like you didn't finish resolving the merge conflicts.

**Comment by kge89** [comment]
Thanks for adding the editor configuration stuff.

File: api/Controllers/CasesController.cs (hunk 1 above)
Comment (tehnools) [reply api/Controllers/CasesController.cs]: I am trying to undo the git pull because i haven't resolved the spacing in vscode yet.

Comment (tehnools) [reply api/Controllers/CasesController.cs]: might have to rebase

**Comment by tehnools** [comment]
> Thanks for adding the editor configuration stuff.

I might have to redo the whole branch again it's just getting too messy

**Pull request closed without merging**